src/Process/ProcessLauncher.php

```diff
@@ -101,12 +105,21 @@ private function restoreSignalHandlers($terminable = true)
         }
     }
 
-    private function run($command)
+    private function run($command, array $arguments)
     {
         if (!function_exists('proc_open')) {
             throw new RuntimeException('The "proc_open" function is not available.');
         }
 
+        $replacements = array();
+
+        foreach ($arguments as $name => $value) {
+            $replacements['%'.$name.'%'] = escapeshellarg($value);
```

Comment:
this does not escape things fully on Windows. See the util class in the Process component